README.md

```diff
@@ -1,2 +1,23 @@
 # Algorithms
 Some algorithms implementation for self-training.
+## Implemented Algorithms
+
+| Algorithm                                                                                         | C#                 |
+|:------------------------------------------------------------------------------------------------- | ------------------ |
+| [Tire](https://github.com/Sharpach/Algorithms/issues/5) [Files](/Algorithms/Algorithms/Trie)| :white_check_mark: | 
```

Comment:
1. There is a typo - "Tire" instead of "Trie"
2. There is also no need in "Files" link